src/core/Track.cpp

```diff
@@ -329,7 +329,7 @@ TrackContentObjectView::TrackContentObjectView( TrackContentObject * tco,
 	m_gradient( true ),
 	m_mouseHotspotHand( 0, 0 ),
 	m_cursorSetYet( false ),
-	m_usesCustomSelectedColor( ! m_tco->useStyleColor() | m_tco->useCustomClipColor() ),
+	m_usesCustomSelectedColor( ! m_tco->usesStyleColor() | m_tco->usesCustomClipColor() ),
```

Comment:
Is there any reason you're not using a short-circuiting or `||`?

Reply:
Oh, I think I intended `||` but typed just one `|` somehow. It works as expected, but I'll change it.